editor/components/rich-text/style.scss

```diff
@@ -80,6 +90,19 @@
 	&.mce-content-body {
 		line-height: $editor-line-height;
 	}
+
+	div.mce-resizehandle {
```

Comment:
Remnant from #5794 where it was designed like this intentionally. I'm don't really have a strong opinion here. @jasmussen what do you think?

Reply:
Last I looked at the inline image branch, the resize handles were those same blue dots we use for regular image blocks. This would be ideal, IMO. Haven't had time to look at this branch, but purely from the discussion, consolidation in design would be 👍 👍

Reply:
Originally I had made the resize handles the same blue design as the image block, and then changed to a white background after this comment from @mtias https://github.com/WordPress/gutenberg/pull/5794#discussion_r185719064. I don't have a strong opinion either way though.